lgc/elfLinker/ElfLinker.cpp

```diff
@@ -301,13 +301,12 @@ ArrayRef<StringRef> ElfLinkerImpl::getGlueInfo() {
 }
 
 // =====================================================================================================================
-// Add a blob for a particular chunk of glue code, typically retrieved from a cache. The blob is not copied,
-// and remains in use until the first of the link completing or the ElfLinker's parent Pipeline being destroyed.
+// Add a blob for a particular chunk of glue code, typically retrieved from a cache.
 //
 // @param glueIndex : Index into the array that was returned by getGlueInfo()
 // @param blob : Blob for the glue code
 void ElfLinkerImpl::addGlue(unsigned glueIndex, StringRef blob) {
-  llvm_unreachable("Not implemented");
+  m_glueShaders[glueIndex]->setElfBlob(blob);
```

Comment:
Is this guaranteed to be in bounds?

Reply:
If it is not there will be an assert in the operator[], so I don't want to add another assert.  It clutters the code.  The comments mention that it has to be an index corresponding to the return results of `getGlueInfo`.  If that is followed, then it is guaranteed to be in bounds.